Editor/ArcadiaProjectInitialization.cs

```diff
@@ -0,0 +1,33 @@
+﻿using UnityEngine;
+using UnityEditor;
+
+namespace Arcadia
+{
+    [InitializeOnLoad]
+    public class ArcadiaProjectInitialization
+        // Starts with an "A" because Editor scripts are initialized alphabetically 
```

Comment:
Unity lets you set script execution order for Monobehaviours via the inspector.  Willing to look into it more if it's an issue.